tools/patch_skia.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# Copyright (c) 2015 Intel Corp. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+"""
+patch_skia.py -- Apply patches to skia when running gclient.
+"""
+
+import os
+import subprocess
+import sys
+
+
+def main(argv=None):
```

Comment:
This block manipulating `argv` isn't necessary, you can just reference `sys.argv` directly.